src/Program/Model/City.hs

```diff
@@ -344,6 +346,10 @@ foreign import javascript unsafe "$r = $3.slice(); $r[$1] = $2;"
 isEmptyCity :: City -> Bool
 isEmptyCity c = collectionLength (objectsIn c) == 0
 
+-- | Filter CityObject properties according to hiddenProperties settings
+shownProps :: City -> CityObject -> JSVal
+shownProps ci obj = Prelude.foldr id (allProps obj) $ deleteProp <$> (hiddenProperties $ cityProperties ci)
```

Comment:
Changes made, but I'm now thinking maybe it's good to have LikeJS instance of "Object Properties"? Also, why there isn't LikeJS instance for (a, b) ?